redis-om-spring/src/main/java/com/redis/om/spring/RedisModulesConfiguration.java

```diff
@@ -109,30 +125,88 @@ public RediSearchIndexer redisearchIndexer(ApplicationContext ac) {
     return new RediSearchIndexer(ac);
   }
 
+  @Bean(name = "djlImageFactory")
+  public ImageFactory imageFactory() {
+    return ImageFactory.getInstance();
+  }
+
+  @Bean(name = "djlImageEmbeddingModelCriteria")
+  public Criteria<Image, byte[]> imageEmbeddingModelCriteria() {
+    return Criteria.builder()
+        .setTypes(Image.class, byte[].class) //
+        .optEngine("PyTorch")  //
```

Comment:
Maybe we can use properties for these values: 
`djl.opt.engine = PyTorch`
`djl.opt.modelUrls = djl://ai.djl.pytorch/resnet18_embedding`